configurations/product/mayastor_config.yaml

```diff
@@ -138,10 +138,12 @@ product:
       2.9.0: "openebs.io/v1beta3"
       2.9.1: "openebs.io/v1beta3"
       2.9.2: "openebs.io/v1beta3"
+      2.9.3: "openebs.io/v1beta3"
```

Comment:
seems a bit repetitive to do this, can't we just assume the version is the same as the previous release? This way we only need to update this when it actually changes